lib/jwt/taskrouter/util.js

```diff
@@ -65,7 +66,99 @@ function defaultEventBridgePolicies(accountSid, channelId) {
   ];
 }
 
+/**
+ * Generate TaskRouter workspace url
+ *
+ * @param {string} [workspaceSid] workspace sid or '**' for all workspaces
```

Comment:
maybe just default it to this if param is undefined so that people can just call `workspacesUrl()`?